tests/numeric_test.py

```diff
@@ -38,15 +38,21 @@ def testSeededRNG(self):
         )
         # Strings (uniformly distributed length)
         self.assertFalse(
-            (ak.random_strings_uniform(1, 10, N) == ak.random_strings_uniform(1, 10, N)).all()
+            (
+                ak.random_strings_uniform(1, 10, N)
+                == ak.random_strings_uniform(1, 10, N)
```

Comment:
similarly i think a lot of these changes are due to `black` without the `--line-length`

Reply:
amanda already pointed this out!